policy/pattern.go

```diff
@@ -1,17 +1,19 @@
 package policy
 
 import (
+	"regexp"
+	"strings"
+
 	"github.com/Masterminds/semver"
 	"github.com/ryanuber/go-glob"
 	"github.com/weaveworks/flux/image"
-	"strings"
-	"regexp"
 )
 
 const (
 	globPrefix   = "glob:"
 	semverPrefix = "semver:"
 	regexpPrefix = "regexp:"
+	regexPrefix  = "regex:"
```

Comment:
These are so similar it's easy to mistake them, and that might lead to mistakes down the road -- maybe `regexpPrefix` and `regexpAltPrefix`?